server/command.go

```diff
@@ -122,12 +123,27 @@ func (p *Plugin) ExecuteCommand(c *plugin.Context, args *model.CommandArgs) (*mo
 			p.postCommandResponse(args, txt)
 			return &model.CommandResponse{}, nil
 		} else if len(parameters) > 1 {
-			features = strings.Join(parameters[1:], " ")
+			optionList := []string{}
+
+			for _, element := range parameters[1:] {
+				if isFlag(element) {
+					flags = append(flags, parseFlag(element))
+				} else {
+					optionList = append(optionList, element)
+				}
+			}
+
+			if len(optionList) > 1 {
+				p.postCommandResponse(args, "Just one option is allowed")
```

Comment:
To be even more clear here I suggest: 
```suggestion
				p.postCommandResponse(args, "Just one list of features is allowed")
```

Given that we defined options = feature list + flags